jwst/tweakreg/tests/test_tweakreg.py

```diff
@@ -21,6 +29,33 @@ def dummy_source_catalog():
     return catalog
 
 
+@pytest.mark.parametrize("inplace", [True, False])
+def test_rename_catalog_columns(dummy_source_catalog, inplace):
```

Comment:
I think we would benefit from adding a docstring-like short statement of the purpose of the test.

Reply:
Thanks! I added docstrings in: 5dc9d4d71